### PR TITLE
Code: Minor fixes.

### PR DESCRIPTION
**Pull request by am11**
//cc @xzyfer.

### CODE REVIEW

File: scripts/install.js [hunk 1]
```diff
@@ -45,13 +47,9 @@ function applyProxy(options, cb) {
...
 
     if (!er) {
-      ['https-proxy', 'proxy', 'http-proxy'].some(function(setting) {
-        var npmProxyUrl = conf.get(setting);
-        if (npmProxyUrl) {
-          proxyUrl = npmProxyUrl;
-          return true;
-        }
-      });
+      proxyUrl = ['https-proxy', 'proxy', 'http-proxy'].filter(function(setting) {
```
Comment (xzyfer) [inline scripts/install.js]: ```proxyUrl = conf.get('https-proxy') || conf.get('proxy') || conf.get('http-proxy');```

Comment (xzyfer) [inline scripts/install.js]: The block below can become 

```
proxyUrl = proxyUrl || env.HTTPS_PROXY || env.https_proxy || env.HTTP_PROXY || env.http_proxy;
```

Comment (am11) [inline scripts/install.js]: Agree that is much better. :+1: 

Fixed by 63331e0.

For better approach in the scenario, where the given array's size is larger, see http://stackoverflow.com/q/28476629/. Lets see what kind of responses we get there! :)

Comment (xzyfer) [inline scripts/install.js]: I use lodash for that https://lodash.com/docs#find

Surprising node-sass is one of the few codebases without lodash or underscore haha

Comment (xzyfer) [inline scripts/install.js]: FYI

```
['https-proxy', 'proxy', 'http-proxy'].reduce(function(prev, cur) { 
  return prev || conf.get(cur); 
}, false);
```

Comment (am11) [inline scripts/install.js]: Yes! underscore definitely provides a great deal of functions when it comes to Array. We can include it in njs projects, but nativity and [vanilla.js](http://vanilla-js.com/) is something I always prefer. :smiley: 

Thanks to node.js to rely on one JS runtime and does not present JS incompatibility issues like those in cross-browser scenarios.

**Pull request merged**